cmd/ls.go

```diff
@@ -22,7 +22,7 @@ import (
 	"text/tabwriter"
 
 	"github.com/dropbox/dropbox-sdk-go-unofficial/files"
-	"github.com/dustin/go-humanize"
+	humanize "github.com/dustin/go-humanize"
```

Comment:
You actually don't need to rename it, you can just use `humanize` without naming it. Go will know where to find the package.